modules/social_features/social_user_export/src/Plugin/UserExportPlugin/UserGroupMemberships.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\social_user_export\Plugin\UserExportPlugin;
+
+use Drupal\group\Entity\Group;
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserGroupMemberships' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_group_memberships",
+ *  label = @Translation("Group memberships (specified)"),
+ *  weight = -198,
+ * )
+ */
+class UserGroupMemberships extends UserExportPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getHeader() {
+    return $this->t('Group memberships (specified)');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getValue(UserInterface $entity) {
+    $group_memberships = Group::loadMultiple(social_group_get_all_group_members($entity->id()));
```

Comment:
Same as above

Reply:
This has been replaced by moving that function to the social_group.helper_service.